kura/org.eclipse.kura.wire.script.filter.provider/META-INF/MANIFEST.MF

```diff
@@ -12,6 +12,7 @@ Import-Package: javax.script,
  org.osgi.service.component;version="1.2.0",
  org.osgi.service.wireadmin;version="1.0.1",
  org.slf4j;version="1.7.0"
+Require-Bundle: org.eclipse.osgi
```

Comment:
Why is this needed?

Reply:
Not needed, maybe accidentally included using the IDE.